### ISSUE TITLE
YouTube video post-processing

### ISSUE THREAD

**Issue by lenucksi** [InnerSourceCommons/InnerSourceLearningPath#214]
We've now got all videos in our hands and I've uploaded them to our YT account. They still need some post-processing for us to set them public.
Here's a copy of [the message containing the remaining tasks](https://innersourcecommons.slack.com/archives/CARTU4XV2/p1581692403087600).

- [x] Check that all titles of the uploads went fine and every video that should be there is there

- [x] Add descriptions to the videos
- [x] Add tags to the videos
- [x] Check if the Playlists are correct and in the correct order. 
- [x]  Check the "this is a authoritative playlist" button on all of them (Set as official series for this playlist)
- [x] Set publishing times on the videos. Probably every video of a series should appear at the same time. This likely triggers the "XYZ uploaded new content" for the people who have subscribed to us "with the bell on" / full upload notification as they say on the videos
- [x] Add links to the articles to the video descriptions
- [x] Ensure that all videos are set to "embeddable"
  - Special checkbox is known to enough people and appears to default to on.
- [x] Ensure that proper license markers as in CC-BY-SA 4.0 are present
- [x] Settings for audience maturity (none) and languages (audio, general: en) are properly set
- [x] Select nice thumbnails for the videos. If the proposed ones are bad one can upload an image (e.g. a video still) for the videos
  - @RJPlog created great suggestions and material here. Splitting this out to its own issue: #243 
- [x] Ping @Tom Sadler (BBC) once there are embeddable links present such that he can update his pull request
  - Only required action here is to update the YT urls to the new non-O'Reilly-URLs
- [x] Come up with Marketing Plans and Publish the results to the respective channels after we ensured that everything is as we want it -> See #60.  Happy to do something there.

**Comment by rrrutledge** [comment]
Before we can work on this we need to finish https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/238 (get the new videos)

**Comment by rrrutledge** [comment]
Some great thoughts from @RJPlog to keep in mind:
* [explanation](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/8cbb02beb41c145ea6bee5235de5c0c206fa8533/assets/thumbnails/thumbnail.asciidoc)
* [example](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/8cbb02beb41c145ea6bee5235de5c0c206fa8533/assets/thumbnails/example5.jpg)

**Comment by RJPlog** [comment]
FYI, I created the thumbnails for the Introduction Playlists. I will continue with the contributor. Don't worry to comment, there is still room for improvement or changes :-)


![Playlist_Thumbnails](https://user-images.githubusercontent.com/44948239/80870089-80b39780-8ca4-11ea-8939-cc91bbd67b0b.jpg)

**Comment by RJPlog** [comment]
please have a look to the new contributors playlist. I also checked titles and added a description, I thought it would be the best way to copy the first chapter of the written description and than add a link to our Homepage.

Comments are welcome :-)

![thumbnails_contributor](https://user-images.githubusercontent.com/44948239/80920231-ae651300-8d6e-11ea-807d-ecc05d4b7cd4.png)

**Comment by RJPlog** [comment]
Hi, after our meeting I made some samples considering your hints. Please leave your comments :-).

- sample #1 primary one
![thumbnail_template_non_serif_headline](https://user-images.githubusercontent.com/44948239/81333821-68c58480-90a5-11ea-8787-8089e481c367.jpg)

**Comment by rrrutledge** [comment]
I like:
* The "InnerSourceCommons.org" from sample no. 1 
* The other text from sample no. 3

I don't think we need the "#01" text - just the segment title should be fine.  e.g. "How does InnerSource work?"

**Comment by RJPlog** [comment]
That's how that would look like (please find the picture after that text being longer than expected :-)):

I have one doubt about letting "#1" out since I do not find a proper way to mark the clustering for the very first video. 

**Why?** We have 4 chapters (Introduction, Product Owner, Trusted Committer, Contributor) with several segments. For all but one we could write like that: 
`<line 3: chapter>` The Trusted Committer
`<line 4: segment>` Ensuring Product Quality

Only the very first would look like that:
`<line 3: chapter>` Introduction
`<line 4: segment>` Introduction

**Proposal 1:** The very first will only be marked with "Introduction" in line 3 like in the sample. All others will have a fourth line, with the name of the segment of the video/article

**Proposal 2:** Only showing line 3 with chapters. The segments names are anyway displayed in the playlist, see my samples in the comments above.

**Proposal 3:** Only showing line 4 with segments. The chapters names are anyway displayed in the playlist, see my samples in the comments above. Hint: this will lead to more than one thumbnail called introduction and conclusion, since these appeare in 3 of the chapters (at the Product Owner they are called opening and closing, but maybe we schould change that.)

Looking at this now, I would vote for Proposal 2. Having a short line with the chapter keeps the Thumbnail better readable, long line for Segment can be seen in playlist anyway.

Ho do you think about?

![thumbnail_template_sample4](https://user-images.githubusercontent.com/44948239/81412951-ea202400-9144-11ea-9ec4-a78f43160440.jpg)

**Comment by rrrutledge** [comment]
I vote for 2 as well.  The segment title will be in the YouTube overlay anyway.  Sounds good.

**Comment by RJPlog** [comment]
this morning I worked on the Product Owner's playlist, hope to find time to rework introduction and contributor playlist with new design in the afternoon

![thumbnails_productowner](https://user-images.githubusercontent.com/44948239/81495119-39ce2f00-92ae-11ea-9045-9d2d67b45333.png)

**Comment by rrrutledge** [comment]
Looks great for thumbnails!

**Comment by rrrutledge** [comment]
Update this file to change the videos shown in the _innersourcecommons.org_ website: https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/config/urls.yaml

**Comment by RJPlog** [comment]
all Points are closed now, I will move this issue to review

**Comment by lenucksi** [comment]
This is actually done, hence closing the issue.
